pkg/varlinkapi/pods.go

```diff
@@ -187,3 +187,35 @@ func (i *LibpodAPI) RemovePod(call iopodman.VarlinkCall, name string, force bool
 
 	return call.ReplyRemovePod(pod.ID())
 }
+
+// GetPodStats ...
+func (i *LibpodAPI) GetPodStats(call iopodman.VarlinkCall, name string) error {
+	pod, err := i.Runtime.LookupPod(name)
+	if err != nil {
+		return call.ReplyPodNotFound(name)
+	}
```

Comment:
should we be vetting that the container is running?

Reply:
Currently GetContainerStats doesn't. If you think it should, both should be updated. WDYT

Reply:
If the container isn't running it just returns all 0s. I'm thinking filtering on running is probably best for both.

Reply:
Fixed